### ISSUE TITLE
求助啊 紧急求助下 还有人吗?

### ISSUE THREAD

**Issue by hexin9456** [jerome2016/Android_USB_UVC_Camera#9]
还有人能回复吗?这个项目编译不了了 有没有人帮帮忙啊!!!谢谢

**Comment by victorwads** [comment]
I Can Try if u still need, here is working normally

**Comment by hexin9456** [comment]
yes，Could you please help me upload my successfully compiled project to GitHub?

**Comment by victorwads** [comment]
yes @hexin9456. u can text me on Telegram, my username on telegram is the same (@victorwads).
